GitUI/Script/ScriptOptionsParser.cs

```diff
@@ -390,6 +400,10 @@ private static string GetRemotePath(string url)
                     break;
 
                 case "cMessage":
+                    newString = EscapeLinefeeds(currentRevision.Body) ?? currentRevision.Subject;
```

Comment:
FWIW, I don't think `module.GetRevision()` loads the body of a commit because it is expensive.

Reply:
Yes, I tried to test it because you had mentioned this. But I trapped into the pit with multiple GE instances overwriting the settings of each other. I will trigger another or modified command if `{cMessage}` is used.